orchestration/dags/data_gcp_dbt/models/analytics/firebase/firebase_home_macro_conversion.sql

```diff
@@ -15,9 +15,9 @@ SELECT
     , COUNT( CASE WHEN booking_timestamp IS NOT NULL THEN 1 ELSE NULL END) AS nb_bookings
     , COUNT( CASE WHEN booking_id IS NOT NULL THEN 1 ELSE NULL END) AS nb_bookings_non_cancelled
     , SUM(delta_diversification) AS total_diversification
-FROM `{{ bigquery_analytics_dataset }}.firebase_home_funnel_conversion`
-LEFT JOIN `{{ bigquery_clean_dataset }}.applicative_database_user` USING(user_id)
-LEFT JOIN `{{ bigquery_analytics_dataset }}.diversification_booking`  USING(booking_id)
+FROM {{ ref('firebase_home_funnel_conversion')}}
+LEFT JOIN {{ ref('int_applicative__user') }} USING(user_id)
```

Comment:
{{ bigquery_clean_dataset }}.applicative_database_user c'est un `select *` de raw_prod.applicative_database_user
VS dans int_applicative__user on a le filtre sur les bénéficiaires (anciens et actuels) 
`where u.user_role in ("UNDERAGE_BENEFICIARY", "BENEFICIARY")` 

```suggestion
LEFT JOIN {{ source('raw','applicative_database_user') }} USING(user_id)
```

Reply:
Oui ! Cette requête permet de justement prendre le user_role ("UNDERAGE_BENEFICIARY", "BENEFICIARY") sinon "Grand Public". 
Du coup c'est mieux d'utiliser la table filtrée correctement